src/components/Navbar.js

```diff
@@ -29,7 +30,11 @@ const Navbar = () => (
           </div>
           <div className="navbar-start">
             {data.allWordpressPage.edges.map(edge => (
-              <Link className="navbar-item" to={edge.node.slug}>
+              <Link
+                className="navbar-item"
+                to={edge.node.slug}
+                key={edge.node.id}
```

Comment:
This is a good fix, suppresses the react warning. I'm not sure that we need to add `id` to the data here though, we can use `slug` for the `key` instead which will (admittedly only slightly) reduce the size of the data required for this page. I'm committing that change directly to master.